Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN go vet ./pkg/... ./cmd/...
 RUN go test ./pkg/... ./cmd/... -coverprofile cover.out
 
 # Build the manager binary
-FROM generate AS build
```

Comment:
not sure it matters, but wanted to make sure the build happened from an image that got `go vet`ted and `go fmt`ted